From 1c614f815ec7f563cbfbab4f8a2155a46080965b Mon Sep 17 00:00:00 2001
From: Elliott Sales de Andrade <quantum.analyst@gmail.com>
Date: Tue, 30 Jun 2020 04:43:57 -0400
Subject: [PATCH 7/7] Increase tolerance for test_usetex and test_rgba_markers.

These should have been updated in images, but weren't. They can be fixed
properly for 3.3.0 final.

Signed-off-by: Elliott Sales de Andrade <quantum.analyst@gmail.com>
---
 lib/matplotlib/tests/test_axes.py   | 2 +-
 lib/matplotlib/tests/test_usetex.py | 3 ++-
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/lib/matplotlib/tests/test_axes.py b/lib/matplotlib/tests/test_axes.py
index 48cfdf582..50dc95925 100644
--- a/lib/matplotlib/tests/test_axes.py
+++ b/lib/matplotlib/tests/test_axes.py
@@ -3435,7 +3435,7 @@ def test_transparent_markers():
     ax.plot(data, 'D', mfc='none', markersize=100)
 
 
-@image_comparison(['rgba_markers'], remove_text=True)
+@image_comparison(['rgba_markers'], remove_text=True, tol=6.2)
 def test_rgba_markers():
     fig, axs = plt.subplots(ncols=2)
     rcolors = [(1, 0, 0, 1), (1, 0, 0, 0.5)]
diff --git a/lib/matplotlib/tests/test_usetex.py b/lib/matplotlib/tests/test_usetex.py
index 25006f174..c2a9c9b20 100644
--- a/lib/matplotlib/tests/test_usetex.py
+++ b/lib/matplotlib/tests/test_usetex.py
@@ -13,7 +13,8 @@ if not mpl.checkdep_usetex(True):
 @image_comparison(
     baseline_images=['test_usetex'],
     extensions=['pdf', 'png'],
-    style="mpl20")
+    style="mpl20",
+    tol=12.9)
 def test_usetex():
     mpl.rcParams['text.usetex'] = True
     fig = plt.figure()
-- 
2.25.4

